sljit_src/sljitNativeARM_32.c

```diff
@@ -1502,8 +1507,16 @@ static SLJIT_INLINE sljit_s32 emit_single_op(struct sljit_compiler *compiler, sl
 	case SLJIT_CTZ:
 		SLJIT_ASSERT(!(flags & INV_IMM) && !(src2 & SRC2_IMM));
 		SLJIT_ASSERT(src1 == TMP_REG1 && !(flags & ARGS_SWAPPED));
+#if defined(SLJIT_CONFIG_ARM_V7) && SLJIT_CONFIG_ARM_V7
 		FAIL_IF(push_inst(compiler, RBIT | RD(dst) | RM(src2)));
 		return push_inst(compiler, CLZ | RD(dst) | RM(dst));
+#else
+		FAIL_IF(push_inst(compiler, RSBS | RD(dst) | RN(src2)));
+		FAIL_IF(push_inst(compiler, AND | RD(dst) | RN(dst) | RM(src2)));
+		FAIL_IF(push_inst(compiler, CLZ | RD(dst) | RM(dst)));
+		FAIL_IF(push_inst(compiler, RSBS | RD(dst) | RN(dst) | 31));
+		return push_inst(compiler, 0x43a00020 | RD(dst)); /* movmi dst, #32 */
```

Comment:
The normal instruction should be used, and the condition codes should be changed. There are some places for this in the code.